fe/fe-core/src/main/java/com/starrocks/persist/OperationType.java

```diff
@@ -591,6 +591,11 @@ public class OperationType {
     @IgnorableOnReplayFailed
     public static final short OP_CLUSTER_SNAPSHOT_LOG = 13513;
 
+    @IgnorableOnReplayFailed
+    public static final short OP_ADD_SQL_QUERY_BLACK_LIST = 13700;
+    @IgnorableOnReplayFailed
+    public static final short OP_DELETE_SQL_QUERY_BLACK_LIST = 13701;
+
```

Comment:
Use 13520 and 13521
These IDs should be used sparingly

Reply:
fixed